core/src/data/value/json.rs

```diff
@@ -123,6 +124,9 @@ mod tests {
             Value::I64(100).try_into(),
             Ok(JsonValue::Number(100.into()))
         );
+        // this doesn't work
+        // assert_eq!(Value::I128(i128::MAX).try_into(), Ok(JsonValue::Number(i128::MAX.into())));
```

Comment:
<https://docs.rs/serde_json/latest/src/serde_json/number.rs.html#28>

Maybe it's because of the size of the number type?

Reply:
If it is okay,  I'll leave it in commented out, since it generates an error.  Maybe someone else can figure it out?

Reply:
I get the following when I try the assert_eq you gave:
left: `Ok(Number(1.7014118346046923e38))`,
 right: `Ok(String("170141183460469231731687303715884105727"))`', core\src\data\value\json.rs:135:9